lib/kino_vega_lite/chart_cell.ex

```diff
@@ -406,4 +439,25 @@ defmodule KinoVegaLite.ChartCell do
       _ -> nil
     end
   end
+
+  defp infer_types({:columns, %{columns: _columns}, data}) do
+    Enum.map(data, fn data -> data |> Enum.at(0) |> type_of() end)
+  end
+
+  defp infer_types({:rows, %{columns: _columns}, data}) do
+    data
+    |> Enum.at(0)
+    |> Enum.map(&type_of/1)
+  end
+
+  defp type_of(data) when is_number(data), do: "quantitative"
+
+  defp type_of(data) when is_binary(data) do
```

Comment:
Could we have number as strings too? Or is that generally a bad idea?

Reply:
It would work in Vega-Lite, but I think nominal is a better guess and the user can override if that's how they store numbers :)